docs/configuring-crash-stats.rst

```diff
@@ -0,0 +1,310 @@
+.. index:: configuring-crashstats
+
+.. _configuring-crashstats-chapter:
+
+Configuring Crash-Stats and PostgreSQL
+======================================
+
```

Comment:
No sections in ```configuring-crashstats``` doc are useful outside of Mozilla. I'll state this at the top of the doc too.